### DISCUSSION TITLE
Please Make Video step by step setup

### DISCUSSION THREAD

**Discussion by jwatan** [engageub/InternetIncome#138] [General]
Hello
sometime read and look that's very different for understanding so it's better way is you must make video for up to end step by step with using this scripts

i hope you solve any problem for me and other which is not using properly

**Comment by engageub** [comment]
Hi,
The code is open source.
The steps are mentioned in [ReadMe](https://github.com/engageub/InternetIncome/blob/main/README.md) file.
You have to follow each step sequentially. 
If you are new to using linux or using operating systems, it might be difficult at the beginning. 

There are only 4 steps involved here after signing up.
1. Install docker
2. Download Script
3. Add your config in `properties.conf`
4. Start the script

If you encounter any difficulties, please let me know which step was problematic. The documentation for that section will be updated accordingly. Additionally, creating a video tutorial is already planned for the next set of updates.

The following screenshot mentions the same.

![image](https://github.com/engageub/InternetIncome/assets/103608400/f83b806a-b377-4ef9-8e8a-6fa78ded8e7d)



Thank you

**Comment by engageub** [comment]
This script was initially created for personal use and was not intended for public distribution. However, if others find it useful, they are welcome to use it. The code is open source, allowing users to download and modify it as needed.

**Comment by jwatan** [comment]
Im using portainer GUI.

please let me know how to setup there..
because i just beginning about docker and as i think that portainer gui is best and easy to use   

i humble request you to make video when you setup in portainer gui.

please reply

**Reply by engageub** [reply]
Hello,
The script does not utilize Portainer and is more lightweight. However, after executing the script, you can view the running containers in Portainer. Could you please let me know which operating system you are using? Ubuntu is recommended for this, and please use the terminal to run the commands.

Thank you

**Comment by jwatan** [comment]
ok im using windows docker gui and need to install ubuntu

**Reply by engageub** [reply]
The steps for installing ubuntu in windows are already mentioned in the [prerequisites]( https://github.com/engageub/InternetIncome?tab=readme-ov-file#house_with_gardenprerequisites).
There is a [youtube video link](https://www.youtube.com/watch?v=2ezNqqaSjq8) mentioned. Please follow the tutorial till 7.30 minutes and then come back to Readme page to download and install the script.

Thank you

**Comment by engageub** [comment]
Hello,
I wanted to follow up and inquire if you were able to successfully run the script.

Thank you
